mmv1/products/pubsub/api.yaml

```diff
@@ -109,7 +109,7 @@ objects:
           For instance, it allows any attached subscription to seek to a timestamp
           that is up to messageRetentionDuration in the past. If this field is not
           set, message retention is controlled by settings on individual subscriptions.
-          Cannot be more than 7 days or less than 10 minutes.
```

Comment:
Thanks for the submission! Would the same be true for the `messageRetentionDuration` field in the subscription resource as well?

Reply:
Nice catch, and I probably should have called this out explicitly, but as far as I understand, this change does not apply to individual subscriptions. You can see it mentioned at the bottom of the [blog post](https://cloud.google.com/blog/products/data-analytics/pubsub-launches-extended-message-storage-for-event-sourcing), and in the console (screenshot) that it still appears to use a 7 day maximum.

<img width="525" alt="Screen Shot 2022-08-18 at 3 36 38 PM" src="https://user-images.githubusercontent.com/7943382/185697711-e96724f8-f661-4d2a-8b95-ab88966d3da9.png">